interfaces/builtin/network_control.go

```diff
@@ -92,6 +92,9 @@ network sna,
 /{,usr/}{,s}bin/wpa_passphrase ixr,
 /{,usr/}{,s}bin/wpa_supplicant ixr,
 
+/dev/rfkill rw,
+/dev/rfkill r,
```

Comment:
Please use 'rw' not 'r' and 'w'. These are equivalent:
```
/dev/rfkill r,
/dev/rfkill w,
```
and `/dev/rfkill rw,`. I can't say why it didn't work for you before, but I suspect it had something to do with the profile not being reloaded or a typo.

+1 with 'rw'.